tests/lib/rules/no-restricted-modules.js

```diff
@@ -99,5 +101,16 @@ ruleTester.run("no-restricted-modules", rule, {
             data: { name: "foo", customMessage: "Please use 'bar' module instead." },
             type: "CallExpression"
         }]
+    }, {
+        code: "require(`fs`)",
+        options: ["fs"],
+        parserOptions: { ecmaVersion: 6 },
+        errors: [{ messageId: "defaultMessage", data: { name: "fs" }, type: "CallExpression" }]
+    }, {
+        // eslint-disable-next-line no-useless-escape
+        code: "require(`crypt\o`)",
```

Comment:
This is escaped `o` in the string `""`, so the code on which the rule will run is still ``require(`crypto`)``

It should be ``code: "require(`crypt\\o`)"`` (we need to escape the backslash)

Reply:
Ahh, I guess I misunderstood original comment then. This is a bit non-sensical example - but it actually shows some examples of cooked vs raw ;) Cooked is "crypto", while raw is "crypt\\o"

Reply:
I'l add proper one, just let me know if this should be removed

Reply:
> I'l add proper one, just let me know if this should be removed

Yes, no need for this one, it's same test as ``code: "require(`crypto`)"``